src/data/simulator.py

```
@@ -109,11 +94,8 @@ def compute_churn_prob(
 ) -> float:
     """Compute effective churn probability for a given month with temporal drift.
 
-    Args:
-        elapsed_months: Number of months elapsed since simulation start (for drift).
-        calendar_month: Actual calendar month (1–12) of the current date (for
-            seasonality).  Falls back to ``elapsed_months + 1`` when omitted so
-            that existing call-sites that pass only three arguments still work.
+    calendar_month: actual calendar month (1–12) for seasonality. Falls back to
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)